Softeq.NetKit.Chat.Application.Services/Services/SystemMessages/ChatSystemMessagesService.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using System.Linq;
+
+namespace Softeq.NetKit.Chat.Application.Services.Services.SystemMessages
+{
+    public class ChatSystemMessagesService : IChatSystemMessagesService
+    {
+        public string FormatSystemMessage(string key, string memberName, string channelName)
```

Comment:
I think it would be better to store fixed string constants for every event type.

Reply:
Done